bwakit/Makefile

```diff
@@ -7,7 +7,8 @@ build: ${build_tool}
 
 ${build_tool}: Dockerfile
 	docker build -t ${name}:${tag} .
-	docker tag -f ${name}:${tag} ${name}:latest
+	-docker rmi ${name}:latest
```

Comment:
I prefer @jpfeil's approach as it is more portable. `tag -f` was removed in 1.12 and i think we should support both.